airbyte-integrations/connectors/source-s3/source_s3/source_files_abstract/stream.py

```diff
@@ -165,143 +158,35 @@ def get_time_ordered_file_infos(self) -> List[FileInfo]:
         """
         return sorted(self.pattern_matched_filepath_iterator(self.filepath_iterator()), key=lambda file_info: file_info.last_modified)
 
-    def _get_schema_map(self) -> Mapping[str, Any]:
-        return_schema: Dict[str, Any] = None
+    @property
+    def _simplified_raw_schema(self) -> Mapping[str, Any]:
         if self._schema != {}:
-            return_schema = deepcopy(self._schema)
-        else:  # we have no provided schema or schema state from a previous incremental run
-            return_schema = self._get_master_schema()
+            return deepcopy(self._schema)
+        return self._auto_inferred_schema
 
-        return_schema[self.ab_additional_col] = "object"
-        return_schema[self.ab_last_mod_col] = "string"
-        return_schema[self.ab_file_name_col] = "string"
-        return return_schema
+    @property
+    def _simplified_schema(self) -> Mapping[str, Any]:
```

Comment:
I'm not sure you need to name these schemas `simplified`.

Reply:
do you have any suggestions how it should be called?

Reply:
_schema?

Reply:
done

Reply:
done